build.sh

```diff
@@ -12,7 +12,7 @@
 SVFHOME=$(pwd)
 sysOS=`uname -s`
 MacLLVM="https://github.com/llvm/llvm-project/releases/download/llvmorg-12.0.0/clang+llvm-12.0.0-x86_64-apple-darwin.tar.xz"
-UbuntuLLVM="https://github.com/llvm/llvm-project/releases/download/llvmorg-12.0.0/clang+llvm-12.0.0-x86_64-linux-gnu-ubuntu-20.04.tar.xz"
+UbuntuLLVM="https://github.com/llvm/llvm-project/releases/download/llvmorg-12.0.0/clang+llvm-12.0.0-x86_64-linux-gnu-ubuntu-16.04.tar.xz"
```

Comment:
Any problem with 20.04?

Reply:
> Any problem with 20.04?
I think the problem is that the latest CTIR only support Ubuntu 18.04 (which needs libtinfo.so.5 ), while the latest LLVM support Ubuntu 20.04 (which needs libtinfo.so.6). See in https://travis-ci.org/github/SVF-tools/SVF/jobs/768682324#L905.

Reply:
I see. it might cause a problem, because, the generated svflib in travis will be published to `npm`. Can some projects inside a ubuntu 20.04 OS still use the svflib built under a 16.04 version?